### PR TITLE
GCP MySQL: remove authorized_network parameter

### PR DESCRIPTION
**Pull request by zucchinidev**
[#183975388](https://www.pivotaltracker.com/story/show/183975388)

Which other branches should this be merged with (if any)?

None

### CODE REVIEW

File: upgrading.html.md.erb [hunk 1]
```diff
@@ -68,6 +68,28 @@ For example:
 For more information about the machine types, see the
 [GCP documentation](https://cloud.google.com/sql/docs/mysql/instance-settings).
 
+#### <a id="gcp-mysql-authorized-network-id"></a> Set `authorized_network_id` for the MySQL service
+
+If you have custom plans already configured for MySQL, and you included the `authorized_network` property,
+you must change the configuration.
```
Comment (jhvhs) [inline upgrading.html.md.erb:74]: **suggestion**: We should tell that the configuration change must happen before starting the upgrade. We should also mention that the same applies to the instances with a user-supplied `authorized_network`. And that all the instances with the attribute left in place will not be manageable after the upgrade.

**Review by jhvhs — CHANGES_REQUESTED** [review]

File: upgrading.html.md.erb (hunk 1 above)
Comment (pivotal-marcela-campo) [reply upgrading.html.md.erb:56]: Good point about instances that have the value already specified. The bad thing is that these attributes are not updateable, so they won't be able to update any of them. I'll make a note for it in the docs and double check with Piyali we are ok with this.

File: reference/gcp-mysql.html.md.erb [hunk 2]
```diff
@@ -166,22 +166,16 @@ provisioning a csb-google-mysql service:
...
-    <td><code>default</code> </td>
-    <td>provision</td>
-  </tr>
   <tr>
     <td><code>authorized_network_id</code></td>
     <td>string</td>
     <td>
       The ID of the Google Compute Engine network to which the instance is connected.
+      It has the format `https://www.googleapis.com/compute/v1/projects/<PROJECT_NAME>/global/networks/<VPC_NAME>`,
+        where the `https://www.googleapis.com/compute/v1/` prefix can be omitted.
```
Comment (jhvhs) [inline reference/gcp-mysql.html.md.erb:175]: **question**: Do we want to mention that this must be the same network PCF compute VMs run on?

File: upgrading.html.md.erb [hunk 3]
```diff
@@ -5,6 +5,66 @@ owner: Cloud Service Broker
...
+## <a id="upgrade-to-1-2"></a> Upgrading from v1.1 to v1.2
+
+### <a id="updating-instances"></a> Update instances to include the proper value for `authorized_network_id`
+
+Make the following changes to existing instances before upgrading.
+
+The old property `authorized_network` was removed, and you have to use `authorized_network_id` instead.
+If you have any instances created passing an explicit value for authorized_network`, these won't be manageable after the upgrade.
+
+See [Set `authorized_network_id` for the MySQL service](gcp-mysql-authorized-network-id) if you have the `authorized_network` property  specified in a custom plan.
```
Comment (jhvhs) [inline upgrading.html.md.erb:17]: Missing `#` in the anchor. (#gcp-mysql...)

**Comment by RichardJJG** [comment]
I've merged this to prevent mounting merge conflicts with other queued PRs for these docs. If further changes are wanted, please raise a new PR.

**Pull request merged**